### PR TITLE
:sparkles: feat Improvements and new developments in the Type One Report.

### PR DESCRIPTION
**Pull request by yecksin**

### CODE REVIEW

File: onecgiar-pr-client/src/app/pages/type-one-report/pages/tor-key-result-story/tor-key-result-story.component.ts [hunk 1]
```diff
@@ -18,59 +21,51 @@ export class TorKeyResultStoryComponent {
   GET_keyResultStoryInitiativeId() {
     this.api.resultsSE.GET_keyResultStoryInitiativeId(this.typeOneReportSE.getInitiativeID(this.typeOneReportSE.initiativeSelected)?.id, this.typeOneReportSE.phaseSelected).subscribe(({ response }) => {
       this.typeOneReportSE.keyResultStoryData = response;
-      console.log(response);
-      //(response);
       this.tablesList = [];
       response.forEach(table => {
+        console.log(table)
```
Comment (gcmartinezg) [inline onecgiar-pr-client/src/app/pages/type-one-report/pages/tor-key-result-story/tor-key-result-story.component.ts:26]: @yecksin is this necessary?

File: onecgiar-pr-client/src/app/pages/type-one-report/pages/tor-key-result-story/tor-key-result-story.component.ts [hunk 2]
```diff
@@ -1,3 +1,6 @@
+/* eslint-disable camelcase */
+/* eslint-disable no-console */
```
Comment (gcmartinezg) [inline onecgiar-pr-client/src/app/pages/type-one-report/pages/tor-key-result-story/tor-key-result-story.component.ts:2]: @yecksin if you are removing line 26, please remove this too

**Review by gcmartinezg — APPROVED** [review]
I have left some comments to be addressed, but in general, approved 👍🏼

File: onecgiar-pr-client/src/app/pages/type-one-report/pages/tor-key-result-story/tor-key-result-story.component.ts (hunk 1 above)
Comment (yecksin) [reply onecgiar-pr-client/src/app/pages/type-one-report/pages/tor-key-result-story/tor-key-result-story.component.ts:26]: Yes, it isn't necessary. I'm going to delete it.

File: onecgiar-pr-client/src/app/pages/type-one-report/pages/tor-key-result-story/tor-key-result-story.component.ts (hunk 2 above)
Comment (yecksin) [reply onecgiar-pr-client/src/app/pages/type-one-report/pages/tor-key-result-story/tor-key-result-story.component.ts:2]: Understood!

**Review by gcmartinezg — APPROVED** [review]

**Pull request merged**